freebites/product
# 46 settings implementation

# Description
Styled Settings page as per the Figma.

Fetched access data from the Settings with live updating checkmarks, implemented popup to allow user to be redirected to Settings app.

*Notifications yet to be implemented*

## Issues
Resolve #46 
Resolve #55 

## Screenshots
![image_123650291 (2)](https://github.com/user-attachments/assets/664001ca-7107-4c93-9b74-bf5600eb8b57)
![image_123650291 (1)](https://github.com/user-attachments/assets/f80f5dbf-3bdf-49fc-92bb-2a8bf2e66c85)

User can click on any part of the Privacy Dropdown to Receive the popup above that can redirect the user to the Settings App

## Test

1.  Open up the Settings tab
2.  Try clicking on every dropdown to see if the dropdown works.
3.  Try clicking on the Account dropdown and clicking on Change Password, Change username, and Log Out to see if they work respectively
4.  Try clicking on Privacy Dropdown for popup and try out both buttons on the popup


## Possible Downsides
*still need to implement notifications*

## Additional Documentations
N/A
